frontend/components/emptyBody.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+class EmptyBody extends React.Component {
```

Comment:
I think 50/50 on my understanding about the use/reusability of this component

Reply:
This component is for when there's nothing going on with this person. Hopefully it will be consistent so players know that there's nothing on this page